package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "@pickware/create-shopify-demo-orders",
+    "version": "0.0.0",
+    "description": "Library to interact with Shopify shops",
+    "repository": "https://github.c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om/pickware/Shopify-DemoStore-Bot",
```

Please also revoke the token.

```suggestion
    "repository": "https://github.com/pickware/Shopify-DemoStore-Bot",
```

---

Done